kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -1,15 +1,15 @@
 define([
     'jquery',
     'bluebird',
-    'kb_common/html',
+    'common/html',
```

Comment:
There should be more changes like this in the next couple PRs. We shouldn't need duplicate `html` modules from different packages.

Reply:
Incidentally, this is the first input widget that I intended to change up for better xSV handling, before I uncovered the validation stuff.

Reply:
I like the versatility of duplication that `kb_common/html` represents -- not only do we duplicate stuff within the narrative codebase, we also have duplicate functionality in other codebases!